### ISSUE TITLE
Issue with latest version of iOS

### ISSUE THREAD

**Issue by thaiat** [testfairy/testfairy-cordova-plugin#6]
The plugin no longer works with iOS9 / XCode latest version

the error is :
```bash
'/testProj6/Plugins/com.testfairy.cordova-plugin/libTestFairy.a(TFGestureRecognizer.o)' 
does not contain bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), 
obtain an updated library from the vendor, 
or disable bitcode for this target. for architecture armv7

clang: error: linker command failed with exit code 1 (use -v to see invocation)

```

**Comment by kpfecteau** [comment]
You can disable bitcode in Xcode and it should compile just fine.  Bitcode is not required by apps yet.

**Comment by gmegidish** [comment]
Fixed in version 1.5.2.
